src/components/Message/MessageSimple/MessageTextContainer.js

```diff
@@ -1,105 +1,97 @@
+  border-top-right-radius: ${({ groupStyle, theme }) =>
     groupStyle === 'rightBottom' || groupStyle === 'rightMiddle'
       ? theme.message.content.textContainer.borderRadiusS
-      : theme.message.content.textContainer.borderRadiusL};
-  padding: 5px;
-  padding-left: 8;
-  margin-top: 2;
-  padding-right: 8;
-  align-self: ${({ alignment }) =>
-    alignment === 'left' ? 'flex-start' : 'flex-end'};
-  border-width: ${({ theme, alignment }) =>
+      : theme.message.content.textContainer.borderRadiusL}px;
+  border-width: ${({ alignment, theme }) =>
     alignment === 'left'
       ? theme.message.content.textContainer.leftBorderWidth
-      : theme.message.content.textContainer.rightBorderWidth};
-  border-color: ${({ theme, alignment }) =>
-    alignment === 'left'
-      ? theme.message.content.textContainer.leftBorderColor
-      : theme.message.content.textContainer.rightBorderColor};
-  background-color: ${({ theme, alignment, type, status }) =>
-    alignment === 'left' || type === 'error' || status === 'failed'
-      ? theme.colors.transparent
-      : theme.colors.light};
+      : theme.message.content.textContainer.rightBorderWidth}px;
+  margin-top: 2px;
+  padding-horizontal: 8px;
+  padding-vertical: 5px;
   ${({ theme }) => theme.message.content.textContainer.css}
 `;
 
-const MessageTextContainer = withTheme((props) => {
+const MessageTextContainer = (props) => {
   const {
-    message,
-    groupStyles = ['bottom'],
     alignment,
-    MessageText = false,
+    groupStyles = ['bottom'],
+    markdownRules = {},
+    message,
+    MessageText,
   } = props;
+  const theme = useContext(ThemeContext);
+
+  if (!message.text) return null;
 
-  const hasAttachment = message.attachments.length > 0 ? true : false;
   const groupStyle =
-    alignment + capitalize(hasAttachment ? 'bottom' : groupStyles[0]);
+    alignment +
+    capitalize(message.attachments.length > 0 ? 'bottom' : groupStyles[0]);
+  const markdownStyles = theme ? theme.message.content.markdown : {};
 
-  if (!message.text) return false;
-  const markdownStyles = props.theme
-    ? props.theme.message.content.markdown
-    : {};
-  const markdownRules = props.markdownRules || {};
   return (
-    <React.Fragment>
-      <TextContainer
-        alignment={alignment}
-        groupStyle={groupStyle}
-        status={message.status}
-        type={message.type}
-      >
-        {!MessageText ? (
-          renderText(message, markdownStyles, markdownRules)
-        ) : (
-          <MessageText {...props} renderText={renderText} />
-        )}
-      </TextContainer>
-    </React.Fragment>
+    <TextContainer
+      alignment={alignment}
+      groupStyle={groupStyle}
+      status={message.status}
+      testID='message-text-container'
+      type={message.type}
+    >
+      {MessageText ? (
+        <MessageText {...props} renderText={renderText} theme={theme} />
+      ) : (
+        renderText({ markdownRules, markdownStyles, message })
```

Comment:
@DanC5 this line break our current custom MessageText component, please note about renderText signature change in CHANGELOG.md as breaking

Reply:
thanks for the heads up, adding to the change log now